docs/zh/navigation.yml

```
@@ -813,7 +813,7 @@ nav:
           - 下载 DCE 5.0 社区版: download/free/dce5-installer-v0.7.0.md
           - 下载 DCE 5.0 商业版: download/business/dce5-installer-v0.7.0.md
           - 子模块和插件:
-              - 下载 Addon 离线包: download/addon/v0.8.0.md
+              - 下载 Addon 离线包: download/addon/history.md
```

```suggestion
              - 下载 Addon 离线包: download/addon/v0.8.0.md
```
左侧 nav 应该是放最新的 addon 吧